helpers/requests/requests.go

```diff
@@ -0,0 +1,87 @@
+package requests
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// ConstructGetURL Constructs a URL to use in a GET request.
+// Queries are passed as a map of string arrays
+func ConstructGetURL(
+	rawURL string, queries map[string][]string, format string, values ...interface{}) (*url.URL, error) {
+
+	urlPath, err := url.Parse(rawURL)
+	if err != nil {
+		fmt.Printf("Unable to parse url %q", rawURL)
+		return nil, err
+	}
+
+	urlPath.Path = fmt.Sprintf(format, values...)
+	var q url.Values = queries
+	urlPath.RawQuery = q.Encode()
+
+	return urlPath, nil
+}
+
+// GetAndParseJSON Makes a request to urlPath with basic auth using user and token.
+//	Attempts to parse the JSON response, returning it in "result" if successful.
+func GetAndParseJSON(result interface{}, user, token string, urlPath *url.URL) error {
+	body, err := getBodyFromRequest(user, token, urlPath.String())
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(body, &result)
+	if err != nil {
+		fmt.Println("Unable to unmarshal refs: ", err)
+		return err
+	}
+
+	return nil
+}
+
+// GetAsString Makes a request to urlPath with basic auth using user and token.
+// Returns the response as a string
+func GetAsString(user, token string, urlPath *url.URL) (string, error) {
+	body, err := getBodyFromRequest(user, token, urlPath.String())
+	if err != nil {
+		return "", err
+	}
+
+	return string(body), nil
+}
+
+func getBodyFromRequest(user string, token string, url string) ([]byte, error) {
```

Comment:
You're always calling this method with an `*url.URL`. Suggest letting the `url` parameter be a `*url.URL` and calling `.String()` inside this function instead